src/vendor/wordpress/_wordpress.scss

```diff
@@ -55,6 +55,55 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: File download button
+ * Applies our pattern library styles to gutenberg file download blocks.
+ */
+
+/**
+ * 1. Prevents stacked file buttons from collapsing on each other. 
+ */
+
+.wp-block-file {
+  + .wp-block-file {
+    margin-top: $wp-button-gap; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying buttons opacity.  
+   */
+
+  .wp-block-file__button,
+  a:not(.wp-block-file__button) {
+    @include button.default;
+    opacity: inherit !important; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying buttons opacity and
+   *    text color.  
+   */
+
+  &.c-button--secondary .wp-block-file__button,
+  &.c-button--secondary :not(.wp-block-file__button) {
+    @include button.secondary;
+    color: inherit !important; /* 1 */
+    opacity: inherit !important; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying buttons opacity and
+   *    text color.  
+   */
+
+  &.c-button--tertiary .wp-block-file__button,
+  &.c-button--tertiary :not(.wp-block-file_button) {
+    @include button.tertiary;
+    color: inherit !important; /* 1 */
+    opacity: inherit !important; /* 1 */
+  }
```

Comment:
🤔 @spaceninja, 

> Would it make sense to merge _this_

When you mention "this", are you referring to the `color` and `opacity` values, or the entire block?

This is how I interpreted your suggestion to consider "merging with the previous block. 

```css
  &.c-button--secondary .wp-block-file__button,
  &.c-button--secondary :not(.wp-block-file__button) {
    @include button.secondary;
  }

  &.c-button--tertiary .wp-block-file__button,
  &.c-button--tertiary :not(.wp-block-file_button) {
    @include button.tertiary;
  }
  
  /**
   * 1. Prevents default WP styling from modifying buttons opacity and
   *    text color.  
   */

  &.c-button--secondary .wp-block-file__button,
  &.c-button--secondary :not(.wp-block-file__button),
  &.c-button--tertiary .wp-block-file__button,
  &.c-button--tertiary :not(.wp-block-file_button) {
    color: inherit !important; /* 1 */
    opacity: inherit !important; /* 1 */
  }
```

However, I don't have a high level of confidence that this is what you intended.

Reply:
You can ignore my comment above, if you'd like. 😉 

Excuse my additional, "think-out-loud" comments on this thread...

I've ended up with this: 
_(These comments do not exist in source)_

```css
.wp-block-file {
  ...

/** 
 * If I don't include `:not(.wp-block-file__button) here, this selector is not specific enough
 * to enforce the desired styles, as rules from the `@wordpress/block-library` override
 * various properties.
 * 
 * I tried the `*` selector here but ran into the same specificity issue.
 */

 .wp-block-file__button,
 :not(.wp-block-file__button) {
    @include button.default;
    opacity: inherit !important; /* 1 */
  }

/** 
 * If I don't include `wp-block-file__button` here, this selector is not specific enough
 * to enforce the desired styles, as rules from the `@wordpress/block-library` override 
 * various properties.
 * 
 * 1. I tried _only_ using the `*` selector here but ran into the same specificity issue.
 * 2. If I'm forced to use `:not(.wp-block-file__button)`, I considered using it here as well 
 *     in hopes of perceived consistency
 */

  &.c-button--secondary {
    .wp-block-file__button,
    * {
      @include button.secondary;
      color: inherit !important;
    }
  }

/** 
 * If I don't include `wp-block-file__button` here, this selector is not specific enough
 * to enforce the desired styles, as rules from the `@wordpress/block-library` override 
 * various properties.
 * 
 * 1. I tried _only_ using the `*` selector here but ran into the same specificity issue.
 * 2. If I'm forced to use `:not(.wp-block-file__button)`, I considered using it here as well 
 *     in hopes of perceived consistency
 */

  &.c-button--tertiary {
    .wp-block-file__button,
    * {
      @include button.tertiary;
      color: inherit !important; 
    }
  }
}
```

which produces the following CSS:

Reply:
That looks great! I'll be honest, in my original comment I intended you to combine the secondary and tertiary blocks, but that was because I missed that they were including different styles. My bad. Your changes here look good.

Reply:
Thanks @spaceninja